docs/usage/list.rst

```diff
@@ -19,6 +19,12 @@ Examples
     -rwxr-xr-x root   root       2140 Fri, 2015-03-27 20:24:22 bin/bzdiff
     ...
 
+    $ borg list /path/to/repo::root-2016-02-15 --pattern "- bin/bz*"
+    drwxr-xr-x root   root          0 Mon, 2016-02-15 17:44:27 .
+    drwxrwxr-x root   root          0 Mon, 2016-02-15 19:04:49 bin
+    -rwxr-xr-x root   root    1029624 Thu, 2014-11-13 00:08:51 bin/bash
+    ...
+
```

Comment:
well, that leaves the user to guess about what "..." will show (and what not).

how about using "- bin/ba*" as pattern so this is more clear?

Reply:
done ec0c3a8a
tnx for the comment.

![image](https://user-images.githubusercontent.com/3264637/97461941-841b9900-193e-11eb-9aa0-b0b92890a58a.png)